RPIPs/RPIP-bounty_incentives.md

```diff
@@ -0,0 +1,120 @@
+Bounty Incentive Levers are OPTIONAL incentives that the pDAO empowers the GMC to use to incentivize the generation and successful completion of bounty proposals.
+* The incentive attached to each lever MAY be set by the GMC.
+* Levers MAY have an absolute and/or a relative component.
+* Incentives for a given bounty MUST NOT reduce the bounty hunter award payout.
+* When incentive levers are modified by the GMC, incentives SHOULD be paid out in such a way that ambiguity favors incentive recipients.
+* The total cost of lever incentives is not limited. The GMC SHOULD set lever incentives with caution.
+
+#### Payout
+* Bounty incentives SHOULD be paid out of the GMC budget.
+* Incentive payouts SHOULD take place within the GMC's regular payout schedule.
+* Incentive payouts MUST be denominated in dollars and SHOULD be paid out in one of the GMC's accepted payout token options, selected by the recipient.
+
+#### pDAO Recourse
+The pDAO MAY use the existing challenge system that is used to challenge GMC decisions following the usual rules. This may take place either:
+* When a lever incentive setting is changed by the GMC.
+* When an incentive is announced to be paid out to a given proposer, definer, or support contact.
+
+### Lever A: Proposal Incentive
+The proposal incentive lever aims to influence the number of bounty proposals that are received by the GMC.
+* This incentive SHOULD be paid to the proposal author for a bounty proposal that is adopted by the GMC.
+* Individual incentive payouts for this lever MAY be reduced at the GMC's discretion based on proposal quality and completeness.
+* Bounty definitions do not need to be included for a proposer to receive this incentive.
+
+### Lever B: Definition Incentive
+The definition incentive lever aims to influence the quality and completeness of the bounty definitions that accompany bounty proposals.
+* This incentive SHOULD be paid to the definition author for bounty definitions provided for a bounty that is adopted by the GMC.
+* This incentive payout SHOULD be paid a maximum of once per proposal, even if the bounty proposal is split into multiple definitions.
+* Individual incentive payouts for this lever MAY be reduced at the GMC's discretion based on definition quality and completeness.
+
+### Lever C: Support Incentive
+The support incentive lever aims to influence the quality and consistency of the bounty work submitted by bounty hunters. It aims to do this by rewarding community contacts that can support, oversee, and assist bounty hunters meet the requirements of open bounties.
+* This incentive SHOULD be split equally between bounty support contacts listed on a bounty where the following conditions are met:
+  1. The bounty MUST be completed (in full or in part) to the satisfaction of the GMC.
+  2. The contact MUST be publicly listed in the bounty definition of the completed bounty.
+  3. The contact MUST NOT be affiliated with the completing bounty hunter.
```

Comment:
What does "be affiliated" mean here? Financial? Non-financial relationship? A strict "appearance of a conflict of interest"? Mostly I ask cuz I think community friendships are quite common and I don't immediately see that as problematic, so my leaning is for the bar be more than that level.

Reply:
Intended to be financial or business relationships, family relationships, friendships pre-dating rocketpool, etc. Not intended to cover the community bonds people have. I will clarify.